cmd/template/cluster/provider/capo.go

```diff
@@ -65,3 +66,40 @@ func WriteOpenStackTemplate(ctx context.Context, client k8sclient.Interface, out
 
 	return nil
 }
+
+func WriteOpenStackTemplateAppCR(ctx context.Context, runner *templateapp.Runner, config ClusterCRsConfig) error {
+	var err error
+
+	clusterAppFlags := &templateapp.Flag{
+		AppName:           config.Name,
+		Catalog:           "control-plane-catalog",
+		InCluster:         true,
+		Name:              "cluster-openstack",
+		Namespace:         fmt.Sprintf("org-%s", config.Organization),
+		Version:           "0.1.0",
```

Comment:
This should most likely be configurable.

Reply:
The`version` is hard coded just for a start and I will add a flag to configure it. And did you mean the other fields as well?

Reply:
Sure, got it. I must also add another `user-configmap` for the app of apps

Reply:
Ok, the fields now have their own flags